haystack/utils/deepsetcloud.py

```diff
@@ -951,6 +1079,23 @@ def get_evaluation_set_client(
         client = DeepsetCloudClient(api_key=api_key, api_endpoint=api_endpoint)
         return EvaluationSetClient(client=client, workspace=workspace, evaluation_set=evaluation_set)
 
+    @classmethod
+    def get_eval_run_client(
+        cls, api_key: Optional[str] = None, api_endpoint: Optional[str] = None, workspace: str = "default"
+    ) -> EvaluationRunClient:
+        """
+        Creates a client to manage eval runs on deepset Cloud.
+
+        :param api_key: Secret value of the API key.
+                        If not specified, will be read from DEEPSET_CLOUD_API_KEY environment variable.
+        :param api_endpoint: The URL of the deepset Cloud API.
+                             If not specified, will be read from DEEPSET_CLOUD_API_ENDPOINT environment variable.
+        :param workspace: workspace in deepset Cloud
```

Comment:
Can we update to: Specifies the name of the workspace for which you want to create the client.